src/Symfony/Component/Notifier/Bridge/Smsapi/SmsapiTransport.php

```diff
@@ -31,18 +31,20 @@ final class SmsapiTransport extends AbstractTransport
 
     private $authToken;
     private $from;
+    private $fast;
 
-    public function __construct(string $authToken, string $from, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    public function __construct(string $authToken, string $from, bool $fast, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
```

Comment:
This is a BC break

Reply:
This class is final, factored by `SmsapiTransportFacotry` should I keep it compatible?

If YES, which option will be better:
A) 
```suggestion
    public function __construct(string $authToken, string $from, bool $fast = false, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
```
or
B) 
```suggestion
    public function __construct(string $authToken, string $from, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null,  bool $fast = false)
```

C) new setter for this option, and keep constructor free of changes

Reply:
I would go with C

cc @fabpot

Reply:
It's a final class, but not an internal one. Let's do C.

Reply:
done @OskarStark @fabpot